openapi-cli/src/test/resources/ballerina-to-openapi/expected_gen/query/query_scenario10.yaml

```diff
@@ -0,0 +1,174 @@
+openapi: 3.0.1
+info:
+  title: PayloadV
+  version: 0.0.0
+servers:
+  - url: "{server}:{port}/payloadV"
+    variables:
+      server:
+        default: http://localhost
+      port:
+        default: "9090"
+paths:
+  /query:
+    get:
+      operationId: getQuery
+      parameters:
+        - name: query0
+          in: query
+          schema:
+            type: string
+            default: ""
+        - name: q1
```

Comment:
is this expected to have `q1` instead of `query1`?

Reply:
It should be `query1`, will fix it